test/test-eth-ledger-bridge-keyring.js

```diff
@@ -369,6 +380,8 @@ describe('LedgerBridgeKeyring', function () {
     const accountIndex = 5
     let accounts = []
     beforeEach(async function () {
+      keyring = new LedgerBridgeKeyring()
```

Comment:
Is this now doing the same thing as the top-level beforeEach? :thinking:

Reply:
yeah good call, removed